RssReader/args_parser.py

```diff
@@ -0,0 +1,63 @@
+"""
+This module process the input arguments
+"""
+
+import argparse
+import re
+import RssReader.logs as logs
+import sys
+
+
+def get_parse(args_in=''):
```

Comment:
Here and below:
It's a good practice to add type hits in your function signatures (e.g. def get_parse(args_in='': str) -> Dict[str, Any]:

Please, take a loot at `typing` lib

Reply:
All lines were found and fixed where appropriate. Will be updated during the next "push" iteration.